packages/cloud-cognitive/src/components/BreadcrumbWithOverflow/BreadcrumbWithOverflow.js

```diff
@@ -281,6 +281,8 @@ export let BreadcrumbWithOverflow = ({
     targetRef: breadcrumbItemWithOverflow,
   });
 
+  console.log(backItem);
```

Comment:
^^ This is probably why you're tests are failing @lee-chase  😄

Reply:
Removed when fixing tests.